apps/pdc-frontend/src/components/MatomoTagManager.tsx

```diff
@@ -0,0 +1,22 @@
+import Script from 'next/script';
+import type { ScriptProps } from 'next/script';
+
+interface MatomoTagManagerProps extends ScriptProps {}
+
+export const MatomoTagManager = ({ nonce, ...restProps }: MatomoTagManagerProps) => {
+  return (
+    <Script
+      id="_next-matomo"
+      src="https://stats.utrecht.nl/analytics/js/container_Ku3GHbr3.js"
+      nonce={nonce}
+      dangerouslySetInnerHTML={{
+        __html: `
+        // initializes the Matomo Tag Manager array
+        const _mtm = window._mtm = window._mtm || [];
+        _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+        `,
```

Comment:
Hier moeten we denk ik iets anders op verzinnen. Volgens de HTML spec 

> If there is a [src](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-script-element.html#attr-script-src) attribute, the element must be either empty or contain only [script documentation](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-script-element.html#inline-documentation-for-external-scripts) that also matches [script content restrictions](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-script-element.html#restrictions-for-contents-of-script-elements).

https://www.w3.org/TR/2011/WD-html5-author-20110809/the-script-element.html

Zouden we het in `onLoad=` ([docs](https://nextjs.org/docs/pages/building-your-application/optimizing/scripts#executing-additional-code)) kunnen zetten?